Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/StorageProviderCommands/Factories/QueryCommandFactoryTest.cs

```diff
@@ -194,75 +157,57 @@ public void CreateForCustomQuery ()
     [Test]
     public void CreateForScalarQuery ()
     {
-      _dataStoragePropertyDefinitionFactoryStrictMock
-          .Setup(mock => mock.CreateStoragePropertyDefinition(DomainObjectIDs.Order1))
-          .Returns(_property1)
+      _dataParameterDefinitionFactoryStrictMock
+          .Setup(mock => mock.CreateDataParameterDefinition(_queryParameter1))
+          .Returns(_dataParameterDefinition1)
           .Verifiable();
-      _dataStoragePropertyDefinitionFactoryStrictMock
-          .Setup(mock => mock.CreateStoragePropertyDefinition(DomainObjectIDs.Order3.Value))
-          .Returns(_property2)
+      _dataParameterDefinitionFactoryStrictMock
+          .Setup(mock => mock.CreateDataParameterDefinition(_queryParameter2))
+          .Returns(_dataParameterDefinition2)
           .Verifiable();
-      _dataStoragePropertyDefinitionFactoryStrictMock
-          .Setup(mock => mock.CreateStoragePropertyDefinition(DomainObjectIDs.Official1))
-          .Returns(_property3)
+      _dataParameterDefinitionFactoryStrictMock
+          .Setup(mock => mock.CreateDataParameterDefinition(_queryParameter3))
+          .Returns(_dataParameterDefinition3)
           .Verifiable();
 
       var commandBuilderStub = new Mock<IDbCommandBuilder>();
       var expectedParametersWithType =
-          new[]
-          {
-              new QueryParameterWithType(
-                  new QueryParameter(_queryParameter1.Name, DomainObjectIDs.Order1.Value, _queryParameter1.ParameterType),
-                  StoragePropertyDefinitionTestHelper.GetIDColumnDefinition(_property1).StorageTypeInfo),
-              new QueryParameterWithType(_queryParameter2, _property2.ColumnDefinition.StorageTypeInfo),
-              new QueryParameterWithType(
-                  new QueryParameter(_queryParameter3.Name, DomainObjectIDs.Official1.ToString(), _queryParameter3.ParameterType),
-                  StoragePropertyDefinitionTestHelper.GetSingleColumn(_property3.SerializedIDProperty).StorageTypeInfo)
-          };
+              new[]
+              {
+                      new QueryParameterWithDataParameterDefinition(
+                              new QueryParameter(_queryParameter1.Name, DomainObjectIDs.Order1, _queryParameter1.ParameterType),
+                              _dataParameterDefinition1),
+                      new QueryParameterWithDataParameterDefinition(_queryParameter2, _dataParameterDefinition2),
+                      new QueryParameterWithDataParameterDefinition(
+                              new QueryParameter(_queryParameter3.Name, DomainObjectIDs.Official1, _queryParameter3.ParameterType),
+                              _dataParameterDefinition3)
+              };
```

Comment:
I think the indent is broken here. should only be two indents / four whitespaces , it got reformatted to double.

Reply:
ReSharper auto-detects 4 spaces of indentation instead of 2 - no idea why. After disabling auto-detection, reformatting did the indentation correctly.